packages/web/src/components/comments/CommentForm.tsx

```diff
@@ -16,12 +16,21 @@ import {
 } from '@audius/harmony'
 import { Form, Formik, useFormikContext } from 'formik'
 import { useSelector } from 'react-redux'
-import { usePrevious } from 'react-use'
+import { usePrevious, useToggle } from 'react-use'
 
+import { DownloadMobileAppDrawer } from 'components/download-mobile-app-drawer/DownloadMobileAppDrawer'
 import { TextField } from 'components/form-fields'
+import { useAuthenticatedCallback } from 'hooks/useAuthenticatedCallback'
+import { useIsMobile } from 'hooks/useIsMobile'
 import { useProfilePicture } from 'hooks/useUserProfilePicture'
 import { audioPlayer } from 'services/audio-player'
 
+const messages = {
+  postComment: 'Post Comment',
+  addComment: 'Add a comment',
+  firstComment: 'Be the first to comment!'
+}
```

Comment:
nit: doesnt need to be part of this PR but I wonder if we should move these into common to share with native

Reply:
yeah i can get that started!